public/app-resources/languages/de.js

```diff
@@ -461,45 +461,45 @@ module.exports = {
-    "The Move Relative step instructs FarmBot to move the specified distance from its current location. For example, if FarmBot is currently at X=1000, Y=1000 and it receives a Move Relative where X=0 and Y=3000, then FarmBot will move to X=1000, Y=4000. If FarmBot must move in multiple directions, it will move diagonally. If you require straight movements along one axis at a time, use multiple Move Relative steps. Move Relative steps should be preceded by a Move Absolute step to ensure you are starting from a known location.": "Der Schritt Relativ-Bewegen weist FarmBot an, die angegebene Entfernung von der aktuellen Position zu verschieben. Wenn FarmBot z. B. derzeit bei den Koordinaten X = 1000, Y = 1000 ist und einen Relativ-Bewegen Befehl mit X = 0 und Y = 3000 ist, wird sich FarmBot zu X = 1000, Y = 4000 bewegen. Wenn sich FarmBot in mehrere Richtungen bewegen soll, wird es sich diagonal bewegen. Wenn du gerade Bewegungen entlang einer Achse gleichzeitig ben�tigst, verwende mehrere Relativ-Bewegen Schritte. Vor einem Relativ-Bewegen Schritt sollte m�glichst ein Absolut-Bewegen Schritt sein, um sicherzustellen, dass der Bot von einem bekannten Ort aus startet. ",
+    "The device has never been seen. Most likely, there is a network connectivity issue on the device's end.": "FarmBot wurde noch nicht gesehen. Wahrscheinlich liegt eine Verbindungsstörung am Gerät vor.",
+    "The Find Home step instructs the device to perform a homing command to find and set zero for the chosen axis or axes.": "Der Schritt "Home finden" weist das Gerät an, die jeweilige Achse zu referenzieren, um Null für die ausgewählte (n) Achse (n) zu finden und einzustellen. ",

```

Comment:
@AscendFB Thanks for the updates!

There is a slight syntax error in the JSON here. The phrase `"Home finden"` must be escaped to `\"Home finden\"`.

Reply:
Thanks, I changed it in my fork and open a PR soon. Looks like its working right now, so I may wait until I found some other Issues.

Reply:
No worries- I patched it on my end. Everything should be visible on my.farm.bot now. :+1: